app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
It's not just duplication. Every time we want to encrypt a new attribute, we'll have to add 3 methods for each attribute, and make sure we named them properly, which can be error-prone. We could end up with a huge class. Eventually, I would like to move this functionality to a gem and allow people to define the attributes they want to encrypt, which is very similar to how the [attr_encrypted](https://github.com/attr-encrypted/attr_encrypted) gem works.

I think it's much easier to be able to define a list of attributes to be encrypted without necessarily knowing how the encryption is implemented. Also, if the implementation ever changes, that means we would need to make a change to 3 methods times the number of encrypted attributes, versus a single change in one place.